doc/Quick-Start-Guide.rst

```diff
@@ -167,6 +167,8 @@ The files will be generated at doc/html/ folder.
 
 
 Tested by:
+- September 20, 2021: Yixuan Li (yixuan.li@seagate.com) in Red Hat Enterprise Linux Server release 7.7 (Maipo)
+
```

Comment:
@kimika88, Please add the git hash here similar to what we see in the line below.

Reply:
5aac28633a149d2c7e6f8d4c502d80dabf7ebb7e